scripts/test_translation_templates.py

```diff
@@ -0,0 +1,150 @@
+from colors import color
+
+import os
+import shutil
+import docker
+import sys
+import subprocess
+
+WELSH_REGIONS = ["wls", "eng"]
+NI_REGIONS = ["nir"]
+
+LANGUAGE_MAP = {"cy": WELSH_REGIONS, "gd": NI_REGIONS}
+
+
+def get_template_content(filename, ignore_context=False):
+    line_beginnings_to_ignore = ['"POT-Creation-Date']
+
+    if ignore_context:
+        line_beginnings_to_ignore += ["#:"]
+
+    with open(filename) as file:
+        return "".join(
+            filter(
+                lambda l: all(
+                    not l.startswith(param) for param in line_beginnings_to_ignore
+                ),
+                file.readlines(),
+            )
+        )
+
+
+def print_filename_results(filename, success=True):
+    if success:
+        print(color(f"{filename} - NO CHANGES", fg='green'))  # noqa: T001
```

Comment:
Can we just use basic logging to avoid the `noqa` throughout the script?

Reply:
The main reason for this was to get coloured output in the script, given that's how we print other successful output in bash scripts.